icu4c/source/data/unidata/create_build_headers.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# © 2021 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+
+# This script runs Bazel to create (generate) header files from data that are
```

Comment:
Not just header files. I suggest renaming this .sh to generate_output_files.sh or simply generate.sh.